### ISSUE TITLE
8.2.6. New - Screen Name/Title: Create a service – Missing Programmatic Label for fieldset

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2456]
Component: Choose order for official languages
Select which language shows first
Observation:
While a `<legend>` is present inside the `<fieldset>` and aria-labelledby is used, the usage is redundant and possibly unnecessary. This could cause verbosity or misinterpretation in some assistive technologies.

(Attachment: 8.2.6. Missing form labels association clarity (form label to radio group).png)

Recommendation: 
You do not need aria-labelledby when the legend is already present and descriptive.
Simplify the markup to:
```
<fieldset>
  <legend>Select which language shows first <span class="sr-only">...</span></legend>
  ...
</fieldset>
```
